src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/personalrelationships/profiledetails/ContactPersonProfileDetailsBookingMovedService.kt

```diff
@@ -7,9 +7,9 @@ import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.data.PrisonerBook
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.nomisprisoner.model.ProfileDetailsResponse
 
 @Service
-class ContactPersonBookingMovedService(
+class ContactPersonProfileDetailsBookingMovedService(
```

Comment:
Renamed and possibly moved profile detail specific classes into its package to avoid confusion with the main contact persons sync